build-parent/pom.xml

```diff
@@ -42,17 +42,18 @@
     <version.jboss-logmanager-ext>1.0.0.Alpha3</version.jboss-logmanager-ext>
     <!-- Weld SE version must be aligned with the version used in WildFly -->
     <version.weld-se>2.4.3.Final</version.weld-se>
+    <!-- Align dependency versions with those present in WildFly parent -->
+    <!-- Doing this explicitly since there is no easy way to import project properties from a pom file -->
+    <version.com.fasterxml.jackson>2.8.9</version.com.fasterxml.jackson>
+    <version.org.javassist>3.20.0-GA</version.org.javassist>
```

Comment:
Using the same property name as is available in WF parent. This way, in case someone builds a Maven plugin that exports (filtered) properties from an imported BOM, we can remove these values as they will be inherited.

Reply:
If the below changes are made, these properties shouldn't need to be added

Reply:
The datatypes being used in the Swagger fraction are not yet available through WildFly 11. So, I shall make the required changes to the Javassist module and test it.